.make/check_gopath.go

```diff
@@ -11,36 +11,45 @@ import (
 	"strings"
 )
 
+func isModuleFlagOn() bool {
+	var moduleFlag = os.Getenv("GO111MODULE")
```

Comment:
We should not put the 1.11 in the environment variable because modules will prevail 1.11. I suggest to rename that variable to
```suggestion
	var moduleFlag = os.Getenv("BUILD_WITH_USE_GO_MODULE_SUPPORT")
```

Reply:
GO111MODULE=on enforces go module behaviour and ignores any vendor directories.

Reply:
@kwk `GO111MODULE=on` env variable is used by go implicitly to ignores traditional way of downloading dependencies(https://github.com/golang/go/wiki/Modules#how-to-define-a-module). Here we are just making use of this env variable to ignore `dep`.

Reply:
Oh, so it's like the GO15VendoreExperiment. Roger that.